File: Backend/src/domain/Entities/Roles.ts
export interface PropRoles{
    id:number;
    roleName: string;
    typeid:number
}

export class Roles{
    public id: number;
    public roleName: string;
    public typeid: number;
    constructor(prop : PropRoles){
        this.id = prop.id;
        this.roleName = prop.roleName;
        this.typeid = prop.typeid;
    }
}

File: BackEnd/Domain/Entities/Roles.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class Roles
    {
        [Key]
        public int id { get; set; }
        public string roleName { get; set; }
        public int typeid { get; set; }
        public Roles()
        {

        }
        public Roles(int id, string roleName, int typeid)
        {
            this.id = id;
            this.roleName = roleName;
            this.typeid = typeid;
        }
    }
}
